docs/part2/20-custom-module.md

```diff
@@ -1,78 +1,79 @@
-2.  The [main.tf](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/main.tf) file which contains the terraform logic calling the landing zone module with the 'right' parameters. The logic is relatively minimal as it mainly delegates the effort to the landing zone module. Two noteworthy aspects:
-
-    a.  The input named `override_json_string` takes the full json definition. In this example, the json passed to the module goes through the templatefile function first to 'inject' the prefix. It is done to ensure uniqueness of the resource group names in the account as indicated in step 1.
-
-    b.  The source is set to the standard vsi pattern and pointing to the latest version at the time this lab was written (v4.4.4).
-
-```hcl
-module "landing_zone" {
-  source           = "git::https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone.git//patterns/vsi?ref=v4.4.4"
-  prefix           = var.prefix
-  region           = var.region
-  ibmcloud_api_key = var.ibmcloud_api_key
-  ssh_public_key   = var.ssh_key
-  override_json_string = templatefile("./override.tftpl", { prefix = var.prefix })
-}
-```
-
-To run the terraform module on your local environment:
-
-1.  Clone the git repository locally with
-    ```bash
+In this step, you run a Terraform module that automates the creation of the custom landing zone topology that you defined in the JSON file in the [previous step](./10-customizing.md).
+
+The objective now is to provide a basic Terraform module that can be easily consumed and that creates the custom VPC topology. You can consider this module as a level of abstraction that simplifies consumption. Consumers of the module do not need to understand the landing zone code or the JSON file that contains the topology definition. They can run a `terraform apply` command with a minimal number of inputs to deploy the fully customized VPC topology in their account.
+
+## Executing the landing zone with a JSON definition
+
+You can find the code for this step in the [custom-slz](https://github.com/IBM/infra-to-app-with-landing-zone/tree/main/custom-slz) directory. The directory contains two important files:
+- The customized JSON file [override.tftpl](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/override.tftpl) file.
+
+    This file is a copy of the JSON definition for our custom topology that you created in the previous step of this lab.
+- The [main.tf](https://github.com/IBM/infra-to-app-with-landing-zone/blob/main/custom-slz/main.tf) file.
+
+    This file contains the Terraform logic that calls the landing zone module with the correct parameters. The logic is relatively minimal because it mostly delegates the effort to the landing zone module.
+
+    Notice these two settings in the `main.tf` file:
+
+    - The `override_json_string` input variable is set to `true`, which instructs the landing zone module to look for and use the `override.json` file in the same directory.
```

Comment:
To me, it's confusing that we say that the var is set to `true`, yet the value doesn't include that boolean. Should we say more about what we're doing in that templatefile?

Reply:
Correct the override_json_string is set to the string value of the templated file that will be used to override the default values.

Reply:
Vincent had - 
The input named `override_json_string` takes the full json definition. In this example, the json passed to the module goes through the templatefile function first to 'inject' the prefix. It is done to ensure uniqueness of the resource group names in the account as indicated in step 1.

Reply:
Got it. I added this info about the placeholders from what's published.

It seems that there might be some differences between the source branch I was using and what's published out there. Hope there aren't many.